cmd/cli/plugin/pinniped-auth/go.mod

```diff
@@ -0,0 +1,69 @@
+module github.com/vmware-tanzu/tanzu-framework/plugin/pinniped-auth
+
+go 1.17
+
+replace (
+	github.com/vmware-tanzu/tanzu-framework/apis/config => ../../../../apis/config
```

Comment:
I don't see usage of this module. Can we remove this if that is the case?

Reply:
Fixed. Thanks